libnmstate/nm/ipv4.py

```diff
@@ -104,15 +104,13 @@ def get_info(active_connection):
     else:
         info['dhcp'] = False
 
+    if info['dhcp']:
```

Comment:
Looks like this has been already asked on line 100, therefore you can place its block there.

Reply:
Right, I didn't consider that the 'else' on line 105 enforces the same behavior.

Done.